projects/organization-management/src/app/store/cost-centers/cost-centers.effects.ts

```diff
@@ -163,7 +173,13 @@ export class CostCentersEffects {
       mapToPayload(),
       mergeMap(payload =>
         this.costCentersService.updateCostCenterBuyer(payload.costCenterId, payload.buyer).pipe(
-          map(costCenter => updateCostCenterBuyerSuccess({ costCenter })),
+          mergeMap(costCenter => [
+            updateCostCenterBuyerSuccess({ costCenter }),
+            displaySuccessMessage({
+              message: 'account.organization.cost_center_management.buyer.update.confirmation',
+              messageParams: { 0: `${payload.buyer.login}` },
```

Comment:
`${payload.buyer.firstName} ${payload.buyer.lastName}` || `${payload.buyer.login}`

to make it work the firstName and lastName has to be added to the CostCenterBuyer object in the method submitCostCenterBuyerForm()  of the cost-center-buyer-edit-dialog.component